roles/jd-server/src/lib/job_declarator/message_handler.rs

```diff
@@ -61,6 +111,13 @@ impl ParseClientJobDeclarationMessages for JobDeclaratorDownstream {
     }
 
     fn handle_declare_mining_job(&mut self, message: DeclareMiningJob) -> Result<SendTo, Error> {
+        {
```

Comment:
redundant brackets

Reply:
@NonsoAmadi10 the redundant brackets remain, I don't see why this comment should be resolved just yet

Reply:
@plebhash i have removed the redundants brackets